prow/githubeventserver/options.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package githubeventserver
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+)
+
+// Options holds all the information needed to create a
+// github event server handler.
+type Options struct {
```

Comment:
The purpose of this is not really clear anymore, I would have expected this to be the only argument to `NewGitHubEventServer`

Reply:
Addressed. These options are now required in `NewGitHubEventServer`